zerver/tests/test_upload.py

```diff
@@ -1789,7 +1799,7 @@ def test_tarball_upload_and_deletion(self) -> None:
         result = re.search(re.compile(r"([0-9a-fA-F]{32})"), uri)
         if result is not None:
             hex_value = result.group(1)
-        expected_url = "https://{bucket}.s3.amazonaws.com:443/exports/{hex_value}/{path}".format(
+        expected_url = "https://{bucket}.s3.amazonaws.com/exports/{hex_value}/{path}".format(
```

Comment:
It's not clear to my why this port number is removed now.  But testing this on the dev server, the download works and contains the contents as expected.